sql/planbuilder/builder.go

```diff
@@ -34,21 +34,23 @@ var BinderFactory = &sync.Pool{New: func() interface{} {
 }}
 
 type Builder struct {
-	ctx             *sql.Context
-	cat             sql.Catalog
-	parserOpts      ast.ParserOptions
-	f               *factory
-	currentDatabase sql.Database
-	colId           columnId
-	tabId           sql.TableId
-	multiDDL        bool
-	viewCtx         *ViewContext
-	procCtx         *ProcContext
-	triggerCtx      *TriggerContext
-	bindCtx         *BindvarContext
-	insertActive    bool
-	nesting         int
-	parser          sql.Parser
+	ctx                 *sql.Context
+	cat                 sql.Catalog
+	parserOpts          ast.ParserOptions
+	f                   *factory
+	currentDatabase     sql.Database
+	colId               columnId
+	tabId               sql.TableId
+	multiDDL            bool
+	viewCtx             *ViewContext
+	procCtx             *ProcContext
+	triggerCtx          *TriggerContext
+	bindCtx             *BindvarContext
+	insertActive        bool
+	insertTableAlias    string
+	insertColumnAliases map[string]string
```

Comment:
can we limit these variables to a single scope? the rest of these objects explicitly outlive a single query

Reply:
Limited it to two scopes (insert scope and combined scope)